### PR TITLE
chore: update cloud icons [DET-7160]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

https://github.com/determined-ai/shared-web/pull/10

## Test Plan

Screen Shot 2022-06-14 at 10 30 51
Screen Shot 2022-06-14 at 10 29 20

adjusted size:
Screen Shot 2022-06-14 at 10 50 51

old
Screen Shot 2022-06-14 at 10 30 58

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hamidzr** [comment]
@alexjdetermined the aws icon on dark theme looks like it could use a better contrast we do we do? a light version?

File: webui/react/src/hooks/useTheme.ts [hunk 1]
```diff
@@ -60,13 +60,13 @@ const updateAntDesignTheme = (path: string) => {
   link.href = `${process.env.PUBLIC_URL}/themes/${path}`;
 };
 
-/*
+/**
  * `useTheme` hook takes a `themeId` and converts the theme object and translates into
  * CSS variables that are applied throughout various component CSS modules. Upon a change
  * in the `themeId`, the hook dynamically updates the CSS variables once again.
  * `useTheme` hook is meant to be used only once in the top level component such as App
  * and storybook Theme decorators and not individual components.
```
Comment (hamidzr) [inline webui/react/src/hooks/useTheme.ts:68]: > hook is meant to be used only once in the top level component such as App
 * and storybook Theme decorators and not individual components.
 
 @hkang1 is this still relevant? this isn't really followed.

File: webui/react/src/components/ResourcePoolCard.tsx [hunk 2]
```diff
@@ -23,11 +26,13 @@ interface Props {
   totalComputeSlots: number;
 }
 
-export const poolLogo = (type: V1ResourcePoolType): React.ReactNode => {
+/** Resource pool logo for based on resource pool type */
```
Comment (hamidzr) [inline webui/react/src/components/ResourcePoolCard.tsx:29]: remove "for"

**Comment by julian-determined-ai** [comment]
> @alexjdetermined the aws icon on dark theme looks like it could use a better contrast we do we do? a light version?

nit: Agreed an icon with a higher contrast would be better.

**Comment by hamidzr** [comment]
> > @alexjdetermined the aws icon on dark theme looks like it could use a better contrast we do we do? a light version?
> 
> nit: Agreed an icon with a higher contrast would be better.

that should be updated now. alex added one to the ticket.

File: webui/react/src/hooks/useTheme.ts (hunk 1 above)
Comment (hkang1) [reply webui/react/src/hooks/useTheme.ts:68]: oh yeah that is no longer true.

Comment (hkang1) [reply webui/react/src/hooks/useTheme.ts:68]: This surfaces some inefficiencies here. A [refactor](https://determinedai.atlassian.net/browse/DET-7551) is in order.

Comment (hamidzr) [reply webui/react/src/hooks/useTheme.ts:68]: deferring to https://determinedai.atlassian.net/browse/DET-7551

**Pull request merged**